### PR TITLE
[nri-kube-events] Use the NRIA_IS_FORWARD_ONLY to avoid entity collision.

### PR DESCRIPTION
**Pull request by gsanchezgavier**
#### Is this a new chart
no
#### What this PR does / why we need it:
Replace the secure forward mode for the forward only mode to avoid entity creation and collision. 
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes https://github.com/newrelic/nri-kube-events/issues/56

#### Special notes for your reviewer:
Staging test [link](https://staging-one.newrelic.com/-/0nVjYPGbXR0)
Also [check](https://staging-one.newrelic.com/-/01qwL5LXgR5) that cluster explorer still has the same functionality regarding the events and linkage to the entities that they belong. 
#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [ ] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[mychartname]`)

### CODE REVIEW

File: charts/nri-kube-events/templates/deployment.yaml [hunk 1]
```diff
@@ -75,6 +75,11 @@ spec:
             - name: NRIA_PROXY
               value: "{{ .Values.proxy }}"
             {{- end }}
+            # Using FORWARD_ONLY mode to avoid the entity creation for the events.
+            - name: NRIA_IS_SECURE_FORWARD_ONLY
+              value: "false"
```
Comment (roobre) [inline charts/nri-kube-events/templates/deployment.yaml:80]: Is it better to set this to `false`, or empty? Maybe @newrelic/caos can provide some insight.

**Review by roobre — APPROVED** [review]
LGTM!

File: charts/nri-kube-events/templates/deployment.yaml (hunk 1 above)
Comment (gsanchezgavier) [reply charts/nri-kube-events/templates/deployment.yaml:80]: my first approach was to set it to empty but i saw an error log on the agent sayin that was not able to convert it to boolean type, so i followed the `false`.

Comment (gsanchezgavier) [reply charts/nri-kube-events/templates/deployment.yaml:80]:
```
level=error msg="unable to interpret environment variables" component=Configuration error="envconfig.Process: assigning NRIA_IS_SECURE_FORWARD_ONLY to IsSecureForwardOnly: converting '' to type bool. details: strconv.ParseBool: parsing \"\": invalid syntax"
```

**Pull request merged**